README.md

```diff
@@ -6,4 +6,50 @@
 
 This repository contains example implementations of the 
 [ExtAuthPlugin interface](https://github.com/solo-io/ext-auth-plugins/blob/master/api/interface.go) and a set 
-of utilities that you can (and should!) use when building your own plugins.
\ No newline at end of file
+of utilities that you can (and should!) use when building your own plugins.
+
+Please refer to the [Auth Plugin Developer Guide](https://gloo.solo.io/dev/writing_auth_plugins/) for an in-depth 
+explanation on how you can use this repository as a template to write your own Gloo Auth plugins.
+
+## Makefile overview
+Following is an overview of the most relevant `make` targets.
+
+### get-glooe-info
+When you are writing your own Ext Auth plugins, you must target a specific GlooE version. This is because of the 
+nature of Go plugins (you can find more info in [this section](https://gloo.solo.io/dev/writing_auth_plugins/#build-helper-tools) 
+of the [Auth Plugin Developer Guide](https://gloo.solo.io/dev/writing_auth_plugins/)). With each release GlooE publishes 
+the information that you will require to replicate its build environment. You can get them by running
+
+```bash
+GLOOE_VERSION=<target-glooe-version> get-glooe-info
```

Comment:
is this actually runnable?  i believe it's only defined in the makefile. perhaps `GLOOE_VERSION=<target-glooe-version> make get-glooe-info`?

Reply:
Oh right, lost the `make` along the way. Good catch!